### PR TITLE
Blog app - controllers specs pull request

### PR DESCRIPTION
**Pull request by theresetuyi**
## Blog app  controllers specs
- add tests on the controllers

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @theresetuyi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Your readme file is missing `key features` and potential `future features` you can add to your project. Kindly add them as they are crucial segments of the readme file template provided by Microverse.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
```
Comment (gemmen29) [inline spec/requests/posts_spec.rb:1]: - [ ] Good job on implementing all requested tests for the `posts#index` action, Kindly also include all tests for the `posts#show` action.

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
```
Comment (gemmen29) [inline spec/requests/users_spec.rb:1]: - [ ] Good job on implementing all requested tests for the `users#index` action, Kindly also include all tests for the `users#show` action.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @theresetuyi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @theresetuyi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**